test/bedrock_Azure_common_kv_test.go

```diff
@@ -31,9 +32,13 @@ func TestIT_Bedrock_AzureCommon_KV_Test(t *testing.T) {
 	backendContainer:= os.Getenv("ARM_BACKEND_STORAGE_CONTAINER")
 	backendTfstatekey:=	k8sName +"-tfstatekey"
 
+	//Copy env directories as needed to avoid conflicting with other running tests
+	azureCommonInfraFolder := "../cluster/test-temp-envs/azure-common-infra-" + k8sName

```

Comment:
Are you leaving the copied test directory or cleaning this up later. Assuming the test is successful the vm will terminate and there is no need. Running locally will continue to create new copied directories even though I see you added this in the git ignore.

Reply:
I'm leaving the copied test directory. When running locally it makes it easier to check and debug.